lib/runtime-core/image-loading-linux-x86-64.s

```diff
@@ -0,0 +1,69 @@
+# NOTE: Keep this consistent with `alternative_stack.rs`.
```

Comment:
Conceptually what is the "alternative stack" and how is it used?

Reply:
`alternative_stack.rs` is mainly those code that works with the OS. Might be a good idea to rename it to something else.